tools/util/injectorutil.go

```diff
@@ -351,7 +351,8 @@ func DeployContract(
 	log.L().Info("Created signed execution")
 
 	injectExecution(selp, execution, client, retryNum, retryInterval)
-	return selp.Hash(), nil
+	selpHash, err := selp.Hash()
+	return selpHash, nil
```

Comment:
return error

Reply:
done

Reply:
Changes done, somehow github is not showing here as Outdated. Please see file in expanded mode.